Equipo2Hosteleria/src/main/java/Controlador/ControladorPanelOperatividad.java

```diff
@@ -0,0 +1,50 @@
+package Controlador;
+
+import Modelo.ConexionMySQL;
+import Modelo.Modelo;
+import Vista.PanelBienvenida;
+import Vista.PanelOperatividad;
+import Vista.Vista;
+
+public class ControladorPanelOperatividad {
+
+	private Modelo modelo;
+	private Vista vista;
+	private Controlador controlador;
+	private PanelOperatividad panelOperatividad;
+	
+	public ControladorPanelOperatividad(Modelo modelo, Vista vista, Controlador controlador) {
+		this.modelo = modelo;
+		this.vista = vista;
+		this.controlador = controlador;	
+	}
+	
+	public void mostrarPanelOperatividad() {
+		this.panelOperatividad = new PanelOperatividad(this);
+		this.vista.mostrarPanel(this.panelOperatividad);
+	}
+	
+	public void accionadoBottonVolverPanelOperatividad() {
+		this.controlador.navegarPanelBienvenida();
+	}
+	
+	public void accionadoBottonPruebaConexionPanelOperatividad() {
+		ConexionMySQL.btnProbarconexionActionPerformed();
```

Comment:
No debemos añadir esta funcionalidad. Lo que podríamos hacer es conectar a la base de datos cuando creemos la instancia del modelo y mostrar un mensaje por la salida estándar.

Reply:
Debido a la inestabilidad de la conexión (Por la librería de MySQL), hemos dejado el botón para poder utilizar el programa aunque no haya conexión, pero si se usa el botón y NO hay conexión, el programa se cierra. Si prefieres que haga esta función automáticamente al entrar, lo cambio, pero podría daros problemas al probarlo si no añadís manualmente la librería, y tenéis la base de datos de nuestro proyecto en MySQL.